src/ServiceControl/Recoverability/Grouping/DomainHandlers/StoreHistoryHandler.cs

```diff
@@ -35,7 +35,7 @@ public void Handle(RetryOperationCompleted message)
                 retryHistory.AddToUnacknowledged(new UnacknowledgedRetryOperation
                 {
                     RequestId = message.RequestId,
-                    RetryType = message.RetryType,
+                    OperationType = message.RetryType,
```

Comment:
Hmm, we can't actually change this class at all or the previously stored documents won't be loadable.